Adding documentation for certificate renewal (#2293)
dapr/docs: daprdocs/content/en/reference/cli/dapr-mtls/_index.md

### Available Commands

```txt

> expiry              Checks the expiry of the root certificate

Reviewer: ```suggestion
expiry              Checks the expiry of the root Certificate Authority (CA) certificate
```

> export              Export the root CA, issuer cert and key from Kubernetes to local files

Reviewer: I notice that the wording on these CLI references is wrong since it says "for Kubernetes" in several places, when it should just say what it does and then then supported platforms tell you where this works.

So this sentence should be 
```suggestion
export              Export the root Certificate Authority (CA), issuer cert and issuer key to local files
```

Reviewer: Can you also apply this approach to the existing export and expiry commands in these files
https://docs.dapr.io/reference/cli/dapr-mtls/dapr-mtls-export/
https://docs.dapr.io/reference/cli/dapr-mtls/dapr-mtls-expiry/

For example in the export file 
1) Change "Export the root CA, issuer cert and key from Kubernetes to local files" to "Export the root Certificate Authority (CA), issuer cert and key to local files"

Reviewer: Actually I have just realized that these export, expiry and renew should all have -k as an option to say that these apply to Kubernetes. No single CLI command should be tied to a single platform environment, so this is a bug we need to fix. Especially when we claim/show this in the docs https://docs.dapr.io/concepts/security-concept/#mtls-in-self-hosted-mode and there are companies who would want to deploy the Sentry service on-premises. In other words it should be this.

dapr mtls export -k -o ./certs
dapr mtls expiry -k 
dapr mtls renew -k

in the same way we have
dapr mtls -k

Reviewer 2: > No single CLI command should be tied to a single platform environment, so this is a bug we need to fix.

There are existing commands which are like that `dapr components -k`, `dapr configurations -k` etc ... (only k8s)
Please create an issue in CLI and this can triaged for future releases. IMO it is a new feature and not a bug ... since the way we query data for self-hosted mode is completely different. 

@msfussell

> renew-certificate   Rotates root CA, issuer cert and issuer key in Kubernetes secret used by Sentry service

Reviewer: ```suggestion
renew-certificate   Rotates the existing root Certificate Authority (CA), issuer cert and issuer key
```

- [`dapr mtls expiry`]({{< ref dapr-mtls-expiry.md >}})
- [`dapr mtls export`]({{< ref dapr-mtls-export.md >}})
- [`dapr mtls renew-certificate`]({{< ref dapr-mtls-renew-certificate.md >}})

### Examples

> 

Reviewer: Can you update the comment in the example below to say "for K8s" like this
```

# Check if mTLS is enabled on the Kubernetes cluster
dapr mtls -k
```
